indicators/base64-encoded-body.yml

```
@@ -0,0 +1,16 @@
+title: Base64-encoded document body
+description: |
+  To evade static analysis, the document body can returned base64 encoded in the response
+  where JavaScript can decode it and append it to the DOM.
+  This helps defeat simple scanners which don't evaluate JavaScript.
+references:
+  - https://urlscan.io/result/3b445b90-5d04-4b36-b29a-cdee4bf9a28e
+  - https://urlscan.io/search/#hash%3A71f119e31661198a11f2c5a24f9a3e01c4b405413efde35a7fb838beb15611df
+detection:
+  documentWriteBase64Unescape:
+    html|contains: "document.write(atob(unescape("
```

Huh, that's weird that they're unescape'ing the string first even though it's not hex encoded 🤔 

If we just look for `document.write(atob(` it'll be a more general purpose rule 🙂 
```suggestion
    html|contains: "document.write(atob("
```

---

Yeah, I found that weird too. Yeah, I agree that would be a better for the purpose of the rule. Thanks!